### PR TITLE
Add Simple Links to Personal Dashboards

### LINKED ISSUE #3997: Add Simple Links - Not sure what category it would fit under.
https://git.fascinated.cc/Fascinated/simple-links

Preview:
Preview

A simple Linktree (and similar) alternative. It is simple to use and lightweight.
If you have any questions, please reply and I will try my best to answer them :)

### PR DESCRIPTION
**Pull request by kokomo123**
This should close #3997 but due to the new requirement of more than 4 months old, this might not make sense to add now and we could put this on hold or i close the PR. 

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [X] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [X] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [ ] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [ ] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and have the license set to `⊘ Proprietary`
- [X] Additions are inserted preserving alphabetical order.
- [X] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [X] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [X] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [X] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [ ] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Miscellaneous`.
- [X] Any software project you are adding to the list is actively maintained.
- [ ] Any software project you are adding was first released more than 4 months ago.
- [X] Any software project you are adding has working installation instructions.
- [X] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1565,6 +1565,7 @@ _Related: [Monitoring](#monitoring), [Bookmarks and Link Sharing](#bookmarks-and
 - [Jump](https://github.com/daledavies/jump) - Yet another self-hosted startpage for your server designed to be simple, stylish, fast and secure. `MIT` `PHP`
 - [LinkStack](https://linkstack.org/) - Open-source, customizable, self-hosted alternative to services like Linktree and Manylink with an intuitive, easy to use user/admin interface. LinkStack allows you to link all your social media platforms easily accessible on one page. ([Demo](https://linksta.cc/), [Source Code](https://github.com/LinkStackOrg/LinkStack)) `AGPL-3.0` `PHP`
 - [Organizr](https://github.com/causefx/Organizr) - Organizr aims to be your one stop shop for your Servers Frontend. `GPL-3.0` `PHP`
+- [Simple Links](https://git.fascinated.cc/Fascinated/simple-links) - Lightweight alternative to Linktree and others. `MIT` `Docker/Nodejs`
```
Comment (KieranRobson) [inline README.md:1568]: I would however remove the `Nodejs` as Docker seems to be the only install method on the git page

Comment (kokomo123) [reply README.md:1568]: Fair enough, will push this now.

**Comment by kokomo123** [comment]
> https://fascinated.cc/ is using the source for this, and it seems to be broken / over enlarged icons, atleast on my machine, until this is fixed / verified to just be on my machine, I would place this on hold
image It looks to be fine for me, perhaps your resolution?

**Comment by nodiscc** [comment]
Initial commit 11 days ago: https://git.fascinated.cc/Fascinated/simple-links/commit/eef0293ede08dd56267319439a8d55d32d4c5bca


> due to the new requirement of more than 4 months old, this might not make sense to add now
>  Any software project you are adding was first released more than 4 months ago.

Correct, it needs to mature and survive the initial 4 months delay.

I considered creating a new label `needs to mature`, and leaving the PR open, but that would leave us with a large number of open PRs that we must recheck from time to time.

Now I think it should be **closed** while still adding the label. This way, anyone could potentially check *closed* issues with this label - but it would not clutter the open PR list - and ask for it to be reopened (or submit a new PR), if the project is still around after the initial delay.

**Pull request closed without merging**